reactor-netty-core/src/test/java/reactor/netty/resources/PoolKeyTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2023 VMware, Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package reactor.netty.resources;
+
+import org.junit.jupiter.api.Test;
+import reactor.netty.resources.PooledConnectionProvider.PoolKey;
+import reactor.netty.transport.AddressUtils;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class PoolKeyTest {
+
+	@Test
+	void test() throws IllegalAccessException {
```

Comment:
nit: Throwing `IllegalAccessException` is not needed

```suggestion
	void test() {
```